backend/capellacollab/projects/toolmodels/modelsources/git/routes.py

```diff
@@ -45,8 +45,8 @@ def verify_path_prefix(db: orm.Session, path: str):
     raise fastapi.HTTPException(
         status_code=status.HTTP_404_NOT_FOUND,
         detail={
-            "err_code": "no_git_instance_with_prefix_found",
-            "reason": "There exist no git instance having the resolved path as prefix. Please check whether you correctly selected a git instance.",
+            "err_code": "NO_GIT_INSTANCE_WITH_PREFIX_FOUND",
+            "reason": "The path validation failed. No Git instance exists with the resolved path as prefix.",
```

Comment:
"The path validation failed" may be a bit redundant in this context. Maybe "No Git instance could be found at the resolved path {whatever resolved path is}"? I guess I'm not sure if they need to know something more specific about that.

Reply:
We do a bit of prefix matching here. For example: 

A Git instance has `https://github.com` as base url. `https://github.com/DSD-DBS` would a valid path (the instance is a prefix of it), but `https://gitlab.com` is not valid.

But the user doesn't really care about it, so I rewrote it a bit: 
```py
detail={
    "err_code": "NO_GIT_INSTANCE_WITH_PREFIX_FOUND",
    "reason": "We couldn't find a matching Git instance. Please that your system administrator allows the given URL.",
    "technical": f"There is no Git instance, which has a prefix of the path '{path}' as URL.",
},
```

Reply:
Ah, ok, that makes more sense. Though unfortunately I think you missed a word in "Please that your system administrator allows the given URL."

Reply:
Replace "Please" with "Make sure".

Reply:
https://github.com/DSD-DBS/capella-collab-manager/pull/1325